### DISCUSSION TITLE
[A-L] (2023/24) Foglio 15 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#289] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d1b9e09c-240f-4b2d-b11e-3b5e8d117060)

**Comment by Elia-Belli** [comment]
Partiamo dall'equazione parametrica trovata in #124, ovvero $U=Span((1,1,1,1),(0,1,0,1))$, e passiamo alla cartesiana come nell'esercizio #126 :

$$\begin{cases}u_1=x_1\\
u_1+u_2=x_2\\
u_1=x_3\\
u_1+u_2=x_4\end{cases}\to\begin{pmatrix}1&0\\
1&1\\
1&0\\
1&1\end{pmatrix}\begin{pmatrix}x_1\\
x_2\\
x_3\\
x_4\end{pmatrix}\to\begin{pmatrix}1&1\\
0&-1\\
0&0\\
0&0\end{pmatrix}\begin{pmatrix}x_4\\
x_3-x_4\\
x_1-x_3\\
x_2-x_4\end{pmatrix}$$

$$U=\\{\underline x\in \mathbb{R}^4| \begin{cases}x_1-x_3=0\\
x_2-x_4=0\end{cases}\\}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/bee0b6be-bd7a-4ea9-aa41-68b305032b96)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d0a0a5e0-b061-413a-8306-2ed3277c02c5)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/16a28796-d216-4ed1-8649-8dca1ca12186)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/eb0fd974-a39c-4663-ae32-8878fc88c775)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-25 172737](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/13dc10f1-9461-4794-931c-4331f6e89d90)
